PyRDF/CallableGenerator.py

```diff
@@ -53,31 +67,65 @@ def _dfs(self, node, prev = 't', ops = []):
 
     def get_callable(self):
         """
-        Function that converts a given
-        graph onto a mapper function and 
-        returns the same
+        Converts a given graph into
+        a callable and returns the same.
+
+        Returns
+        -------
+        function
+            The callable function that traverses
```

Comment:
callable function is perhaps redundant, I would just leave either function or callable.